python/src/tests/helpers/test_find_cell_ids_in_same_hierarchy.py

```diff
@@ -1,8 +1,11 @@
 import json
-import os
 
 from worker.helpers.find_cell_ids_in_same_hierarchy import (
-    find_all_cell_ids_in_cell_sets, find_cell_ids_in_same_hierarchy)
+    find_all_cell_ids_in_cell_sets,
+    find_cell_ids_in_same_hierarchy,
+)
+
+cells_data_file = "tests/data/FindCell.json"
```

Comment:
Wouldn't having the path hard-coded like this instead of concatenating it using `os.path.join()` expose this to the possibility of having platform-specific path errors?

Reply:
Good point Agi, although the tests seem to run ok in Github actions

Reply:
I will switch back to using .join, it is safer